### DISCUSSION TITLE
Assignation automatique de la durée des vidéos/audios

### DISCUSSION THREAD

**Discussion by Klelian** [skarab42/marv#29] [Ideas]
Pour les animations:
- Dans la time line, faire en sorte que le logiciel détecte automatiquement la longueur de la vidéo/son/fig qu'on lui fournit et la mette par défaut (quitte à la réduire après au besoin)
- et également peut etre par défaut aussi la taille de l'animation à celle du canvas

**Comment by skarab42** [comment]
+1 pour la durée des vidéos/audios

Pour la taille des assets je suis pas fan, je préfère avoir la taille original par défaut. 
Par contre je peu ajouter un bouton "fit screen".
